### PR TITLE
AFKR-1: Added configuration for identifier types and identifier sources

### PR DESCRIPTION
**Pull request by rubailly**
- Added `identifier_sources_void.csv` to retire all default Bahmni identifier sources

- Added `identifierSource.csv` which override parent source file and creates easydoc identifier sources

- Added `PatientIdentifierTypes-1.zip`, a metadata sharing package that retires default Bahmni inditifier types and creates new ones for easydoc.

### CODE REVIEW

File: configuration/globalproperties/clinical_gp.xml [hunk 1]
```diff
@@ -0,0 +1,115 @@
+<config>
+  <globalProperties>
+    <globalProperty>
+      <property>emrapi.sqlSearch.activePatients</property>
+      <value>select distinct
```
Comment (rbuisson) [inline configuration/globalproperties/clinical_gp.xml]: @rubailly why is this file added? Does it differ from the Haiti one?

File: configuration/globalproperties/i18n_gp.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<config>
```
Comment (rbuisson) [inline configuration/globalproperties/i18n_gp.xml]: Same remark here. What is different from the Haiti file?

File: configuration/personattributetypes/easyDocPersonAttributeTypes.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+Uuid,Void/Retire,Name,Description,Format,Foreign uuid,Searchable,Edit privilege,_order:1000
+29f58586-e331-4da1-a558-3d4210e48664,,impl.email,Email,java.lang.String,,,,
```
Comment (rbuisson) [inline configuration/personattributetypes/easyDocPersonAttributeTypes.csv]: Let's name this **additionalPersonAttributeTypes.csv**

**Review by rbuisson — CHANGES_REQUESTED** [review]

File: configuration/personattributetypes/easyDocPersonAttributeTypes.csv (hunk 3 above)
Comment (rubailly) [reply configuration/personattributetypes/easyDocPersonAttributeTypes.csv]: [Done](https://github.com/mekomsolutions/openmrs-config-easydoc/pull/4/commits/739bc2911b24e1e526238e835f2875c32df30a21)

File: configuration/globalproperties/clinical_gp.xml (hunk 1 above)
Comment (rubailly) [reply configuration/globalproperties/clinical_gp.xml]: [Done](https://github.com/mekomsolutions/openmrs-config-easydoc/pull/4/commits/be61da316052fc30be7d24f34667c66cbd27b74c)

File: configuration/globalproperties/i18n_gp.xml (hunk 2 above)
Comment (rubailly) [reply configuration/globalproperties/i18n_gp.xml]: [Done](https://github.com/mekomsolutions/openmrs-config-easydoc/pull/4/commits/be61da316052fc30be7d24f34667c66cbd27b74c)
